CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
+    for global artifacts and downloads.
+  * Configure `NERVES_DATA_DIR` to `XDG_DATA_HOME` if defined.
+  * Add `nerves` config for `source_date_epoc` to enable reproducible builds.
+  * Allow `erlinit.config` options to be overridden from Mix config.
+    The options from the system are merged with the options specified in the
+    config for `:nerves, :erlinit`.
```

Comment:
* `erlinit.config` options can be overridden using the application config now. For example, in your `config.exs` you can now add:

```
config :nerves, :erlinit,
  ctty: "ttyAMA0"
```

I think we need to update the docs for this feature. I couldn't find any documentation except for the source code. Here's what we currently say: https://github.com/nerves-project/nerves/blob/553de3a139402ed47b7ba307eb1379ab017f0225/docs/Advanced%20Configuration.md#overwriting-files-in-the-root-filesystem

Reply:
I added a blurb about it here https://github.com/nerves-project/nerves/blob/553de3a139402ed47b7ba307eb1379ab017f0225/docs/Advanced%20Configuration.md#overriding-erlinitconfig-from-mix-config

Maybe we need to move it above the section mentioned?